### PR TITLE
ARC-792: update README

### PR DESCRIPTION
**Pull request by thombergs**
Updating the README with some additional information and formatted according to content design.

The rendered version of the README can best be viewed here: https://github.com/atlassian/github-for-jira/tree/ARC-792-update-readme#readme

### CODE REVIEW

**Review by yvonnevictoria — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,112 +2,157 @@
...
+Existing users of Jira's built-in DVCS connector that meet the [requirements](https://github.com/atlassian/github-for-jira/blob/main/README.md#requirements) should migrate to this integration. If you've not yet been prompted to do so, you can manually kick off the migration by:
+
+1. Sign in to your Jira Cloud account.
+2. From the left sidebar in Jira, select **Jira Settings > Applications > DVCS accounts**.
+3. Follow the prompt to upgrade your GitHub connection.
 
-## Questions? Need help?
-Take a look through the [troubleshooting steps in our support guide](SUPPORT.md).
+## Need help?
+Take a look through the troubleshooting steps in our [support guide](https://github.com/atlassian/github-for-jira/blob/main/SUPPORT.md).
```
Comment (mboudreau) [inline README.md:150]: You don't need full path of repo, just the local file

File: README.md [hunk 2]
```diff
@@ -2,112 +2,157 @@
...
 
-## Contributing
-Want to help improve the integration between GitHub and Jira? Check out the [contributing docs](CONTRIBUTING.md) to get involved.
+## Contribute
+Want to help improve the integration between GitHub and Jira? Check out the [contributing docs](https://github.com/atlassian/github-for-jira/blob/main/CONTRIBUTING.md) to get involved.
```
Comment (mboudreau) [inline README.md:153]: Same here.

File: README.md [hunk 3]
```diff
@@ -2,112 +2,157 @@
...
 
 ## License
-The project is available as open source under the terms of the [MIT License](LICENSE).
+The project is available as open source under the terms of the [MIT License](https://github.com/atlassian/github-for-jira/blob/main/LICENSE).
```
Comment (mboudreau) [inline README.md:156]: And here.

File: README.md [hunk 4]
```diff
@@ -2,112 +2,157 @@
...
-2. From the left sidebar in Jira, select Jira Settings -> Applications -> DVCS accounts.
-3. Follow the prompt to upgrade your GitHub connection
+### See GitHub CI/CD data in Jira
+GitHub Actions is a feature from GitHub for automation such as CI/CD. If you’re setting this up for the first time, follow [GitHub Actions Documentation - GitHub Docs](https://docs.github.com/en/actions). If you already have GitHub Actions and want to see CI/CD data from Github in Jira, include the Jira issue key in your commit message, branch name, or PR.
+
+### How the integration works
+When a workflow (e.g. GitHub Action) or development event (e.g. pull request, commit, branch) runs, our app receives a webhook from GitHub. We then extract the issue key from the respective branch/commit/PR and send this information to Jira.
+
+## Migrate from the DVCS Connector
+Existing users of Jira's built-in DVCS connector that meet the [requirements](https://github.com/atlassian/github-for-jira/blob/main/README.md#requirements) should migrate to this integration. If you've not yet been prompted to do so, you can manually kick off the migration by:
```
Comment (mboudreau) [inline README.md:143]: Fairly sure you can change the link to just be `#requirements` so it doesn't reload the page.

File: README.md [hunk 5]
```diff
@@ -2,112 +2,157 @@
...
+5. Once installation completes you will be redirected to [Atlassian Marketplace](https://marketplace.atlassian.com/apps/1219592/github-for-jira?hosting=cloud&tab=overview) to install the GitHub for Jira app.
+6. From there, follow the instructions above to [install from Atlassian Marketplace](#install-from-atlassian-marketplace).
+
+## Configure app
+
+### Connect your GitHub organization to Jira
+As part of the installation flow, you should be directed to a **GitHub configuration** screen to configure your GitHub 
+organizations in the Jira app.
+
+> :warning: If you are not directed, navigate to the Apps page of your Jira instance and click **Configure integration** under the ”GitHub” heading. If you do not see this, click on **Manage apps** and **Get started** under GitHub for Jira. This will bring you to the app's configuration screen.
```
Comment (atrigueiro) [inline README.md:82]: Maybe it would be good to state here that only Jira admins can see the `Manage your apps` link.

File: README.md [hunk 6]
```diff
@@ -2,112 +2,157 @@
...
 
-This is useful if your installation is setup to send Development information to an instance you no longer have access to, or to audit instances other admins in your org may have previously configured. This only gives you the permission to delete the connection, and will not give you access to the instance itself. You will still need to be granted access in Jira if you want to be able to view the Development information that's been sent to that instance.
+### See GitHub CI/CD data in Jira
+GitHub Actions is a feature from GitHub for automation such as CI/CD. If you’re setting this up for the first time, follow [GitHub Actions Documentation - GitHub Docs](https://docs.github.com/en/actions). If you already have GitHub Actions and want to see CI/CD data from Github in Jira, include the Jira issue key in your commit message, branch name, or PR.
 
-## Migrating from the DVCS connector
+### How the integration works
+When a workflow (e.g. GitHub Action) or development event (e.g. pull request, commit, branch) runs, our app receives a webhook from GitHub. We then extract the issue key from the respective branch/commit/PR and send this information to Jira.
```
Comment (bgvozdev) [inline README.md:140]: nit: maybe "the app", not "out app"?

File: README.md (hunk 5 above)
Comment (atrigueiro) [reply README.md:82]: nit: apparently it changed from `Manage apps` to `Manage your apps`
Screen Shot 2021-11-16 at 9 18 01 am

File: README.md (hunk 6 above)
Comment (bgvozdev) [reply README.md:140]: nit: maybe "We extract" -> "The app extracts" ?

File: README.md [hunk 7]
```diff
@@ -2,112 +2,157 @@
...
 
-#### Jira Permission Scopes
+#### Jira permission scopes
 Read, Write, and Admin for Development Information (branches, commits, and pull requests)
 
-#### GitHub Permission Scopes
+#### GitHub permission scopes
 
-|Permission scope|Why we need it|
+|**Permission scope**|**Why we need it**|
```
Comment (bgvozdev) [inline README.md:105]: nit: we -> the app ?

**Review by mboudreau — DISMISSED** [review]
mmmm documentation...

**Review by bgvozdev — DISMISSED** [review]

File: README.md (hunk 7 above)
Comment (thombergs) [reply README.md:105]: I see you want to distance yourself from the app... noted :)

**Review by mboudreau — APPROVED** [review]

**Review by bgvozdev — APPROVED** [review]

**Pull request merged**